src/EnvelopeCreator/DefineEnvelope.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DocusignBundle\EnvelopeCreator;
+
+use DocuSign\eSign\Model;
+use DocusignBundle\EnvelopeBuilder;
+use Symfony\Component\Routing\Router;
+use Symfony\Component\Routing\RouterInterface;
+
+class DefineEnvelope
+{
+    private $router;
+
+    public function __construct(RouterInterface $router)
+    {
+        $this->router = $router;
+    }
+
+    public function handle(EnvelopeBuilder $envelopeBuilder): void
+    {
+        $envelopeBuilder->setEnvelopeDefinition(new Model\EnvelopeDefinition([
+            'email_subject' => EnvelopeBuilder::EMAIL_SUBJECT,
```

Comment:
This const should be in the current class instead of the EnvelopeBuilder, as it's used here